spec/ibandit/iban_spec.rb

```diff
@@ -1698,6 +1698,107 @@
     end
   end
 
+  describe "valid_australian_details" do
+    subject { iban.valid_australian_details? }
+
+    context "with non-Australian details" do
+      let(:arg) do
+        {
+          country_code: "GB",
+          bank_code: "1234",
+          branch_code: "200000",
+          account_number: "55779911",
+        }
+      end
+
+      it { is_expected.to be(true) }
+    end
+
+    context "with Australian details" do
+      let(:arg) do
+        {
+          country_code: "AU",
+          branch_code: "123-456",
+          account_number: "123456789",
+        }
+      end
+
+      context "without a modulus checker defined" do
+        it { is_expected.to be(true) }
+      end
+
+      context "with a modulus checker defined" do
+        before do
+          Ibandit.modulus_checker = double(
+            valid_branch_code?: valid_branch_code,
+          )
+        end
+        after { Ibandit.modulus_checker = nil }
+        before { iban.valid_australian_details? }
```

Comment:
Should this be the `subject`? If not can we collapse it into the `before` block above it